src/main/java/codeu/controller/ServerStartupListener.java

```diff
@@ -25,6 +32,13 @@
   public void contextInitialized(ServletContextEvent sce) {
     try {
       List<User> users = PersistentStorageAgent.getInstance().loadUsers();
+
```

Comment:
Update the comment on this contextInitialized method, since we're doing more than just loading data now.